### ISSUE TITLE
[Keyboard Navigation - Home]: The setting and delete button are not accessible via keyboard under the "Interpolation Mode" pane.

### ISSUE THREAD

**Issue by natellium** [grafana/grafana#66244]
### User Experience: 
Keyboard-dependent users will not be able to access the controls and will not be able to perform the required action on the controls If interactive controls present on any page or pane are not keyboard accessible, hence fail to use their functionality.  Note: User credentials should NOT be included in the bug.

Repro Steps:
Open #Url: https://play.grafana.org/   Sign in with valid credentials. "Home" page appears. Navigate the Feature Showcase pane. Select any one of the data such as,Time Series Graphs". "Visualizations" page opens, Navigate to the "Interpolation Mode" pane and expand it. Select the "Delete" button. Verify the issue.

### Actual Result: 
The setting and delete button are not accessible via keyboard under the "Interpolation Mode" pane.

### Expected Result:
 The setting and delete button should be accessible via keyboard under the "Interpolation Mode" pane.
Note:  Same issue repro for the "Remove operation" button of "Click to show description" popup present in the "Explore" page.

### Severity
`SEV2`

**Comment by khushijain21** [comment]
Hello, I'd like to work on this, can you direct to the file location?

**Comment by polibb** [comment]
Is this issue about the "Interpolation mode" row or each of the panels in the row? In both cases I cannot reproduce an a11y issue, maybe it's fixed by the new panel header (?).

https://github.com/grafana/grafana/assets/13227501/113b6c9d-ab54-4228-9f79-16bb67959578

In regards to
>Note: Same issue repro for the "Remove operation" button of "Click to show description" popup present in the "Explore" page.

I don't understand where in the Explore page is "Click to show description".

**Comment by natellium** [comment]
@polibb Same here, wasn't able to reproduce the first issue, might have been solved with the new panel header 🤷‍♀️ 

As for the Explore part, it took a bit of looking into the code but I think I found it. I think it may be specific to the Prometheus DS and it's similar to the issue we have with the panel menu not being keyboard accessible as it only sows on hover. 

https://github.com/grafana/grafana/assets/74534993/949b7c5b-b6a6-40d2-9fa7-0cb4518f2a6a

You can see in the video the controls next to the Sum operation are only keyboard accessible once you hover. I'd say we can close this issue and create a new one for Explore only, wdyt?

**Comment by polibb** [comment]
@natellium Ahaa, I see. That's true,here's a separate issue for this field: https://github.com/grafana/grafana/issues/69388. By "Explore page" I didn't understand the query editor, but the Explore section in Grafana (in main navigation). 

P.S. The menu not being visible on hover is not an issue right now, but I did notice that when a panel is selected through keyboard navigation, pressing "e" on the keyboard does not open Panel Edit. Whereas pressing "e" while hovering on any panel in a dashboard does open Panel Edit for that panel. Issue for it: https://github.com/grafana/grafana/issues/69390
